docs/higher-order/tca.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<p>Suggested markdown:</p><blockquote><br></blockquote><blockquote>This given run already provide interesting insights with regards to the neuronal activity during repeats of the natural movies.</blockquote><blockquote>Our movie was shown in 2 blocks : Block 0 and 1, each made of successive repeats of the movie. These blocks are identified on the previous plot with a yellow dashed line. TCA identifies several components that are block specific and sometime transiently activated at the onset of the first or second block.</blockquote><blockquote>Certain components are evenly distributed across all trials, are non-homogenously distributed across cells (in VISp, VISl, VISpm) and show very strong transient patterns of activity. Those components could be related to specific aspects of the movie.</blockquote><blockquote><br></blockquote><blockquote>Overall, this decomposition can be used to refine further analysis of each components and test hypothesis on their respective roles. TCA will also naturally exclude global modes of activity in its decomposition and allow to isolate local components from global components.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>